cylc/rose/utilities.py

```diff
@@ -442,3 +439,15 @@ def simplify_opts_strings(opts):
             seen_once.append(item)
 
     return ' '.join(reversed(seen_once))
+
+
+def dump_rose_log(dest_root, node):
+    dumper = ConfigDumper()
+    timestamp = DateTimeOperator().process_time_point_str(
+        print_format='%Y%m%dT%H%M%S%Z'
+    )
+    fname = f'log/{timestamp}-rose-suite.conf'
```

Comment:
Shouldn't this be:
```suggestion
    fname = f'log/conf/{timestamp}-rose-suite.conf'
```
See https://github.com/cylc/cylc-rose/issues/10#issuecomment-762802718

Reply:
Looks like it.